src/instructlab/train/linux_train.py

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Standard
+import psutil # to determine system memory
```

Comment:
It's a 3rd party import.

Please add the requirement to `requirements.txt`, too. Right now it's a transient dependency from `peft`.

Reply:
Thanks, I have added it to requirements.txt